src/services/validation.js

```diff
@@ -22,9 +24,11 @@ export async function validateSRN(userID, validateCount, isSingleEntryOnly, redi
     //this condition checks the second time, since still not valid, just changes the flag and continues with the plio.
     else if (!doesUserExist && validateCount == 1) {
     validateCount = 2;
```

Comment:
Can you document this at the top of the function.. that the function sets `validateCount = 0 if this, 1 if this and so on`